crates/runtime/Cargo.toml

```diff
@@ -52,6 +52,7 @@ arrow_sql_gen = { path = "../arrow_sql_gen", optional = true }
 bb8 = {workspace = true, optional = true}
 bb8-postgres = {workspace = true, optional = true}
 base64 = "0.22.0"
+keyring = "2.3.2"
```

Comment:
Can we make this an optional feature?

Reply:
Added `keyring` feature to runtime, and enabled by default for spiced for now